scripts/verify-correctness.js

```diff
@@ -74,6 +115,15 @@ module.exports = async callback => {
         // If the last equation holds, the inverse trade must be profitable as well
       })
     )
+
+    // TODO: extract addresses of traded tokens from previous checks
+    const tradedTokenAddresses = []
```

Comment:
To @josojo: I think it's easier to modify your code to yield the list of traded token than to implement my own fetching here, also because it would probably mean that I have to reread all orders again just to find the token ids of the orders.